emmet-core/emmet/core/vasp/task.py

```diff
@@ -152,6 +160,8 @@ def entry(self) -> ComputedEntry:
             "energy": self.output.energy,
             "parameters": {
                 "potcar_spec": self.input.potcar_spec,
+                "is_hubbard": self.input.is_hubbard,
```

Comment:
Can we "compute" this from the `hubbards` field rather than relying on it being right in the DB?

Reply:
probably not since the hubbard U parameters and the actual U values are set independent of each other in VASP.  So unless there is some testing of doing ` LDAU=.TRUE.` calculations with zero U values I'm not 100% sure VASP doesn't do some change to the energy.

Reply:
I mean, isn't `is_hubbard  = len(hubbards) > 0 ` ?